src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CriteriaField.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.ccd.domain.model.definition;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+public class CriteriaField  implements Serializable {
```

Comment:
there's an extra space here.
Also, what about we call this class something like SearchFilter?

Reply:
removed the space. 
This is a parent class for search and workbasket (I'm not sure if that name is a good choice) that's why I didn't want to make is specific for search but went on with a more generic name